docs/GCSFile-batchsource.md

```diff
@@ -25,6 +25,11 @@ Properties
 ----------
 **Reference Name:** Name used to uniquely identify this source for lineage, annotating metadata, etc.
 
+**Use Connection** A boolean indicates whether to use a connection, if a connection is used, 
+the credentials does not need to be provided.
+
+**Connection** The connection name, should have the macro function ${conn:(connection-name)} to provide.
```

Comment:
Is there a drop-down for connection?

Reply:
Yes, UI will show a page to let user choose the connection

Reply:
We don't include 'The' or 'A' in the first sentence of these descriptions.

Name of the connection to use. Project and service account information will be provided by the connection.